terraform/gitlab/gitlab.tf.json.template.py

```diff
@@ -1669,11 +1674,27 @@ def qq(*words):
                                         }
                                     }
                                 }
-                            }, indent=4)
+                                # FIXME: Hitting user_data size limits
+                                #        https://github.com/DataBiosphere/azul/issues/5314
+                            })
                         },
                     ],
+                    # A reboot is required to ensure that FIPS mode is in effect
+                    'power_state': {
+                        'mode': 'reboot',
+                        'message': 'Rebooting in order to enable FIPS mode'
```

Comment:
```suggestion
```

unless message is required. If it is, set it to something generic. 

Either the message or the comment above is redundant. And the message is of no use to anyone currently logged in. They don't need to know why the reboot is happening, just that it is. Then again, no one is going to be logged in anyways, that shortly after the instance comes up.

Reply:
Not required.
>… [the default is None.](https://cloudinit.readthedocs.io/en/18.3/topics/examples.html#reboot-poweroff-when-finished)